test/generic/genericShowInstruction_test.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:cie_team1/generic/genericShowInstruction.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+@Timeout(const Duration(seconds: 20))
+void main() {
+  group('Introduction', () {
+    testWidgets('1 widgetTest for introduction', (WidgetTester tester) async {
+      // Tells the tester to build a UI based on the widget tree passed to it
+      await tester.pumpWidget(
+        new StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return new MaterialApp(
+              home: new Material(
+                  child: new Column(
+                    children: <Widget>[
+                      GenericShowInstruction.showInstructions(null, context),
+                    ],
+                  )
+              ),
+            );
+          },
+        ),
+      );
+
+      final Iterable<Widget> listOfWidgets = tester.allWidgets;
+
+      int counter = 0;
+      for (Widget widget in listOfWidgets) {
+        if (widget is Text) {
+          if (counter == 0) {
+            expect(widget.data,"Instructions");
+            counter++;
+          } else if (counter == 1) {
+            expect(
+                widget.data,"Unfortunately you have not downloaded any course data "
```

Comment:
A little bit dirty to just write the text here. I will accept, but please open an Issue and request changes.
Text should be stored, for example in static variables. So you could use it here without writing the same...

Reply:
okay :)